tests/testthat/test-stat-bin.R

```diff
@@ -172,13 +172,10 @@ comp_bin <- function(df, ...) {
 
 test_that("inputs to binning are checked", {
   dat <- data_frame(x = c(0, 10))
-  expect_snapshot_error(comp_bin(dat, breaks = letters))
-  expect_snapshot_error(bin_breaks_width(3))
-  expect_snapshot_error(comp_bin(dat, binwidth = letters))
-  expect_snapshot_error(comp_bin(dat, binwidth = -4))
-
-  expect_snapshot_error(bin_breaks_bins(3))
```

Comment:
No longer testing `bin_breaks_width(3)` and `bin_breaks_bins(3)` because these are always called from inside `compute_bins()`, which protects against improper ranges.